### PR TITLE
Lemminx: Add version 0.19.1-541

### PR DESCRIPTION
**Pull request by marcolongol**
- [X] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Hi!

Please separate the refactoring in a different PR.

**Comment by marcolongol** [comment]
Done!

**Comment by rashil2000** [comment]
https://github.com/redhat-developer/vscode-xml/releases

Download from here. It has more recent releases.

**Comment by rashil2000** [comment]
You removed autoupdate

**Comment by marcolongol** [comment]
... and that's why you don't review PR's at 4AM

File: bucket/lemminx.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+    "version": "0.21.0",
+    "description": "LeMMinX is a language server for XML and XSD files",
+    "homepage": "https://github.com/redhat-developer/vscode-xml",
```
Comment (rashil2000) [inline bucket/lemminx.json:4]:
```suggestion
    "homepage": "https://github.com/eclipse/lemminx",
```

We can keep this as homepage as this is the original source

**Pull request merged**